README.md

```diff
@@ -1125,18 +1125,24 @@ Options for the [OpenID Connect](http://developer.okta.com/docs/api/resources/oi
 
 ### Smart Card IdP
 
-**:information_source: EA feature:** The Smart Card IdP feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea). This feature is only available for okta-hosted widget implementations.
+**:information_source: EA feature:** The Smart Card IdP feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea).
 
 Settings for authentication with a Smart Card `X509` type IdP.
 
+- `certAuthUrl` *(required)* - [MTLS SSO Endpoint](https://developer.okta.com/docs/reference/api/idps/#mtls-single-sign-on-sso-endpoint-object) that requests the browser for a certificate and performs authentication.
```

Comment:
I think this should read: 

- `certAuthUrl` *(required)* - [The `url` property of the MTLS SSO Endpoint Object](https://developer.okta.com/docs/reference/api/idps/#mtls-single-sign-on-sso-endpoint-object). The browser prompts the user to select a client certificate when it attempts to access this URL.

Reply:
Changed wording a little bit.. Let me know what you think.

Reply:
LGTM!